tests/unit/kvs/kvs_rparams_test.c

```diff
@@ -208,7 +208,7 @@ MTF_DEFINE_UTEST_PRE(kvs_rparams_test, cn_compact_vblk_ra, test_pre)
     ASSERT_EQ((uintptr_t)ps->ps_stringify, (uintptr_t)param_default_stringify);
     ASSERT_EQ((uintptr_t)ps->ps_jsonify, (uintptr_t)param_default_jsonify);
     ASSERT_EQ(256 * 1024, params.cn_compact_vblk_ra);
-    ASSERT_EQ(0, ps->ps_bounds.as_uscalar.ps_min);
+    ASSERT_EQ(MPOOL_MBLOCK_SIZE_DEFAULT, ps->ps_bounds.as_uscalar.ps_min);
```

Comment:
I think we need checks here to ensure the default value is within bounds, because in this case I think it would have failed...